File: benchmarks/poly/jsonserialize/jsonserialize.rb
_t0 = Process.clock_gettime(Process::CLOCK_MONOTONIC, :nanosecond)
parts = []
400000.times do |i|
  parts << '{"id":' + i.to_s + ',"name":"user' + i.to_s + '","score":' + (i % 100).to_s + '}'
end
out = parts.join("\n")
puts out.length
puts parts.length
$stderr.puts "bench_ns=#{Process.clock_gettime(Process::CLOCK_MONOTONIC, :nanosecond) - _t0}"

File: benchmarks/poly/jsonserialize/jsonserialize.js
const _t0 = process.hrtime.bigint();
const parts = [];
for (let i = 0; i < 400000; i++) {
  parts.push('{"id":' + String(i) + ',"name":"user' + String(i) + '","score":' + String(i % 100) + '}');
}
const out = parts.join("\n");
console.log(out.length);
console.log(parts.length);
console.error(`bench_ns=${process.hrtime.bigint() - _t0}`);
